config/fxdata/magic.cfg

```diff
@@ -67,7 +67,7 @@ Name = SPELL_REBOUND
 CastAtThing = 0
 ShotModel = NOSHOT
 Duration = 250
-SelfCasted = 1 37
+SelfCasted = 1 926
```

Comment:
Is 926 really appropriate? It sounds like something has hit the shield rather than the shield being conjured.

Reply:
I don't know. It does sound like that, but at least it does communicate a purpose of the spell. I doubted it too (along with a few of the others), tried it a bit ingame, and decided to just put them all in and let you guys give feedback.

Reply:
Looks like we may need to search for a free sound effect for this.....

Reply:
I don't have any idea what I would want this to sound like,.... but there's plenty: https://opengameart.org/art-search-advanced?keys=&field_art_type_tid%5B%5D=13&sort_by=count&sort_order=DESC